include/http/request_handler.js

```diff
@@ -24,6 +24,7 @@ var async   = require('async');
 var domain  = require('domain');
 var Cookies = require('cookies');
 var util    = require('../util.js');
+var _ = require('lodash');
```

Comment:
Here is a link to it.  https://lodash.com/  It is mainly used for formatting collections such as removing duplicate values as we do in the request_handler on line 780.

"themesToCheck = _.uniq(themesToCheck);"

Ben Tidwell
Software Engineer III | Talent Network
CareerBuilder
5550-A Peachtree Pkwy, Norcross GA 30092

From: Brian <notifications@github.com<mailto:notifications@github.com>>
Reply-To: pencilblue/pencilblue <reply@reply.github.com<mailto:reply@reply.github.com>>
Date: Sunday, July 12, 2015 at 7:33 PM
To: pencilblue/pencilblue <pencilblue@noreply.github.com<mailto:pencilblue@noreply.github.com>>
Cc: Ben Tidwell <ben.tidwell@careerbuilder.com<mailto:ben.tidwell@careerbuilder.com>>
Subject: Re: [pencilblue] Multi Tenancy (#680)


In include/http/request_handler.js<https://github.com/pencilblue/pencilblue/pull/680#discussion_r34427098>:

> @@ -24,6 +24,7 @@ var async   = require('async');
>  var domain  = require('domain');
>  var Cookies = require('cookies');
>  var util    = require('../util.js');
> +var _ = require('lodash');


For my lack of familiarity with lodash, what does it by us here?

-
Reply to this email directly or view it on GitHub<https://github.com/pencilblue/pencilblue/pull/680/files#r34427098>.